src/main/fc/fc_core.c

```diff
@@ -84,6 +84,9 @@
 #include "flight/pid.h"
 #include "flight/servos.h"
 
+#ifdef SPEEDYBEEF4
```

Comment:
We have put a lot of work into removing all target specific defines from everywhere except for target definitions. Reintroducing them is not an option, this will have to be solved in a generic way. Since we are going to move towards generic targets for >= F4, the solution should not only be generic, but should also be driven by runtime config, instead of compile time defines. I can see this solved as a generic GPIO pin config that is bound to an 'event' (in this case arming, but could be extended to other events later).

@martinbudden, @blckmn, @jflyper, @ledvinap thoughts?

Reply:
Agreed. We should look to making everything generic, and operate out of config.

Reply:
What about making it 'switch device', outside of target?